packages/plugin-trpc/web/src/lib/index.ts

```diff
@@ -0,0 +1,2 @@
+export * from './handler';
+export { z } from 'zod';
```

Comment:
zod is entirely transient here, just reexporting `z` which is required for creating the types

Reply:
Sounds reasonable to me. We can easily extend that should developers want access to more `zod` functionality.